test/v6/ukPropertyBsas/retrieve/def1/model/response/RetrieveUkPropertyBsasFixtures.scala

```diff
@@ -32,7 +32,20 @@ object RetrieveUkPropertyBsasFixtures {
       |  "requestedDateTime": "2000-01-01T10:12:10Z",
       |  "adjustedDateTime": "2000-01-01T10:12:10Z",
       |  "taxableEntityId": "AA999999A",
-      |  "taxYear": 2021,
+      |  "taxYear": 2024,
+      |  "status": "valid"
+      |}
+      |""".stripMargin
+  )
+
+  val downstreamNonTysMetadataJson: JsValue = Json.parse(
+    """
+      |{
+      |  "calculationId": "f2fb30e5-4ab6-4a29-b3c1-c7264259ff1c",
+      |  "requestedDateTime": "2000-01-01T10:12:10Z",
+      |  "adjustedDateTime": "2000-01-01T10:12:10Z",
+      |  "taxableEntityId": "AA999999A",
+      |  "taxYear": 2020,
```

Comment:
If you need to split out chunks of JSON (or anything else) because something is changing (it looks it is just the tax year that is important here, but it isn't obvious) then it would be clearer here, and at the call site, if a method rather than a val were used. (Also cuts down on duplication.)

(Also for SE, but you've not done this for foreign retrieve.)